src/pint/models/expdip.py

```diff
@@ -0,0 +1,286 @@
+"""Exponential dips due to, e.g., profile change events in J1713+0747"""
+
+from typing import List, Optional, Union
+import numpy as np
+import astropy.units as u
+from astropy.time import Time
+from pint.models.parameter import floatParameter, prefixParameter
+from pint.models.timing_model import DelayComponent
+from pint.toa import TOAs
+
+
+class SimpleExponentialDip(DelayComponent):
+    """Simple chromatic exponential dip model for events like the profile changes
```

Comment:
Can you actually sketch out the form here?  Not necessarily all of the math, although you could include it for RTD rendering.  But the basic form?  Otherwise things like the index don't make sense.